packages/core/saved-objects/core-saved-objects-migration-server-internal/src/zdt/utils/get_alias_actions.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { AliasAction } from '../../actions';
+
+interface GetAliasActionOpts {
+  indexPrefix: string;
+  currentIndex: string;
+  existingAliases: string[];
+  kibanaVersion: string;
+}
+
+/**
+ * Build the list of alias actions to perform, depending on the current state of the cluster.
+ */
+export const getAliasActions = ({
+  indexPrefix,
+  currentIndex,
+  existingAliases,
+  kibanaVersion,
+}: GetAliasActionOpts): AliasAction[] => {
+  const actions: AliasAction[] = [];
+
+  const globalAlias = indexPrefix;
+  const versionAlias = `${indexPrefix}_${kibanaVersion}`;
```

Comment:
So, I decided to keep the version alias for now. 

Main reason is that the SOR is accessing the indices via these aliases at the moment:

https://github.com/elastic/kibana/blob/34c228b0c941308e01298e31223e7af0a5a2cee9/packages/core/saved-objects/core-saved-objects-base-server-internal/src/utils/get_index_for_type.ts#L18-L25

And I didn't want to handle this problem right now. 

We can revisit / think about it when we'll perform the SOR changes in https://github.com/elastic/kibana/issues/150312